src/lib/services.json

```diff
@@ -0,0 +1,17 @@
+{
+  "github": {
+    "tokens": {
+      "token": "ONEGRAPH_GITHUB_TOKEN"
```

Comment:
Here `"token"` is the name of the property exposed on the `secrets` object. There could in theory be multiple tokens per service: pass them as keys in the `"tokens"` object.